ymer/src/main/java/com/avanza/ymer/PersistedInstanceIdStatistics.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer;
+
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.openspaces.core.util.ConcurrentHashSet;
```

Comment:
Just wanted to check whether it is intentional that a gs collection is used here instead of a collection type from the jdk?

Reply:
No, that was a mistake 😓  I switched to using a collection from jdk instead.

Reply:
👍